### PR TITLE
Added Mechanical Module 

### PR DESCRIPTION
**Pull request by FieryPhoenix909**
# Description
> Adds a new Mechanical Module to the game with mechanically themes ships. It currently consists of a small, medium, large, and capital sized ship as well as one new gun.

### CODE REVIEW

**Comment by GooeyHub** [comment]
Can one of the admins please verify this patch?

**Comment by vampcat** [comment]
@GooeyHub: ok to test

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by vampcat** [comment]
Okay, so I just tested the ship.. and it feels weird. It does not point towads my mouse - probably because of the origin not being correctly set in Box2D. You'll have to trace the boundary again, but set the origin to (0.5, 0.5) (center of image) before tracing the mesh.

**Comment by FieryPhoenix909** [comment]
Oh, ok. Also, which two files am I using 2-spaces in?

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by FieryPhoenix909** [comment]
The edits have been made. Please let me know if anything else needs to be changed.

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by FieryPhoenix909** [comment]
I have added a large ship and a new weapon to the mechanical module.

File: engine/src/main/resources/settings.ini [hunk 1]
```diff
@@ -2,8 +2,8 @@ x=1366
 y=768
 fullscreen=false
 controlType=1
-sfxVol=1.0
-musicVol=0.25
+sfxVol=0.0
```
Comment (Malanius) [inline engine/src/main/resources/settings.ini]: Please don't commit your settings :)

Comment (vampcat) [reply engine/src/main/resources/settings.ini]: Definitely afree with Malanius - please revert this file.

Comment (FieryPhoenix909) [reply engine/src/main/resources/settings.ini]: Sorry, that was an accident!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Pull request merged**